src/clj_http_hystrix/core.clj

```diff
@@ -74,14 +77,6 @@
       (warn exception message)
       (warn message))))
 
-(defn status-codes
```

Comment:
@joelittlejohn it looks like this wasn't being used anywhere, so i'm proposing to remove it. though maybe its an expected api function of this library?